3723/3/db/migrate/20190726093200_create_users.rb

```diff
@@ -0,0 +1,11 @@
+# :reek:FeatureEnvy
+# :reek:UncommunicativeVariableName
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :username
```

Comment:
If you have validation within the model will good decision to add such in the database
```ruby
t.string :username, null: false # because you have `validates :username, presence: true`
```